src/lib.rs

```diff
@@ -46,15 +37,13 @@ impl Task for AsyncReflink {
 // Async version
 #[napi(js_name = "reflinkFile")]
 pub fn reflink_task(src: String, dst: String) -> AsyncTask<AsyncReflink> {
-    let src_path = PathBuf::from(src);
-    let dst_path = PathBuf::from(dst);
-    AsyncTask::new(AsyncReflink { src: src_path, dst: dst_path })
+    AsyncTask::new(AsyncReflink { src, dst })
 }
 
 // Sync version
 #[napi(js_name = "reflinkFileSync")]
 pub fn reflink_sync(env: Env, src: String, dst: String) -> Result<JsNumber> {
```

Comment:
I am not familiar with napi, can `reflink_sync` here can take `&str` as type for `src` and `dst`?

Reply:
I think it has to be `String`, maybe I'm missing something though https://napi.rs/docs/concepts/values.en#string